cumulus/parachains/runtimes/testing/penpal/src/xcm_config.rs

```diff
@@ -188,7 +196,7 @@ impl Contains<Location> for ParentOrParentsExecutivePlurality {
 pub struct CommonGoodAssetsParachain;
```

Comment:
haha some leftover `CommonGoodAssetsParachain` :D, I would aslo rename this to `SystemParachains`,
and also I think we have common type `pub struct AllSiblingSystemParachains`